pandas/tests/indexes/timedeltas/test_timedelta.py

```diff
@@ -509,6 +509,10 @@ def test_fields(self):
         tm.assert_series_equal(s.dt.seconds, Series(
             [10 * 3600 + 11 * 60 + 12, np.nan], index=[0, 1]))
 
+        # preserve name (GH15589)
```

Comment:
might be better to add something to 

pandas/tests/indexes/datetimelike.py. These are inherited by all of the datetimelike test indexes.